### DISCUSSION TITLE
"Prompt User" Action

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#7741] [Ideas]
Scenario : 
I have a button that does something. 
I would like to prompt the user, before continuing. Similarly to the Delete Row or Execute Query.
Maybe there should be an Action "Prompt User" which can then be followed by a Continue If / Stop If.
The "Prompt User" action, should have a bindable "message" property.

Example : 
Button -> OnClick
  -> Action "Prompt User". Message "Are you sure you want to add all {{ State.totalIngedients }} ingredients to the Order ?"
  -> Action "Continue If". Continue If Step 1.Response === "OK" 
  -> Action "Execute Query". ( Would be nice to be able to disable / customize default notifications for success or failure )
  -> Action "Notification" -> "Ingredients Added" ( Or "Operation Failed. Ingredients were not added to the order" )

Thoughts ?

**Comment by melohagan** [comment]
This would be a really nice addition! I've turned this discussion into a feature request: https://github.com/Budibase/budibase/issues/7762
